packages/vite-node/src/utils.ts

```diff
@@ -199,3 +199,21 @@ function traverseBetweenDirs(
     longerDir = dirname(longerDir)
   }
 }
+
+export function createImportMetaEnvProxy() {
+  // packages/vitest/src/node/plugins/index.ts:146
+  const booleanKeys = [
+    'DEV',
+    'PROD',
+    'SSR',
+  ]
+  return new Proxy(process.env, {
+    get(_, key) {
+      if (typeof key !== 'string')
+        return undefined
+      if (booleanKeys.includes(key))
+        return !!process.env[key]
+      return process.env[key]
+    },
+  })
```

Comment:
With this implementation I have a question: if you set `import.meta.env.SSR = false` will it be `false` or `true`? Since `process.env` casts a value to a string, I would assume that `process.env.SSR === 'false'` which will be `true` when casted back to boolean

Reply:
I think it's correct now.